sdk/storage/storage-blob/src/ContainerClient.ts

```diff
@@ -1363,6 +1363,7 @@ export class ContainerClient extends StorageClient {
     if (options.includeUncommitedBlobs) {
       include.push("uncommittedblobs");
     }
+    if (options.prefix === "") options.prefix = undefined;
```

Comment:
Can we make this 2 lines? 

Recommend to have prettier vsc extension, which will format automatically.

Reply:
I have prettier, it didn't complain about the inline-if. I've updated to a block now.